core/services/job/util.go

```diff
@@ -29,6 +29,19 @@ func EVMChainForJob(job *Job, set evm.ChainSet) (evm.Chain, error) {
 	return chain, nil
 }
 
+// EVMChainIDForJob parses the job spec and retrieves the evm chain id found
+func EVMChainIDForJobSpec(spec *OCR2OracleSpec) (chainID int64, err error) {
```

Comment:
Careful! EVM chain IDs can in theory be larger than int64. Would suggest using `utils.Big` for this type instead (we already do this everywhere else in the code)